config/migrations/20220511133621-add-publication-report-types.ttl

```diff
@@ -0,0 +1,54 @@
+@prefix mu: <http://mu.semte.ch/vocabularies/core/> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix ext: <http://mu.semte.ch/vocabularies/ext/> .
+@prefix pub: <http://mu.semte.ch/vocabularies/ext/publicatie/> .
+
+@prefix PublicationReportTypeScheme: <http://themis.vlaanderen.be/id/concept-scheme/b75a85c3-a599-4d4f-a2a2-dea583f2fd86> .
+@prefix publicationReportType: <http://themis.vlaanderen.be/id/concept/publicatierapporttype/> .
+
+# PublicationReportTypes are only used to make the /publicaties/overzicht/rapporten route
+#   act more like any other route: fetching a (list of) models
+#   Every PublicationReportType stands for a certain combination of a type to group by and types to filter on.
+#   The definition of the filters and group options these types are done in the frontend.
+
+pub:Publicatierapporttype a ext:PublicClass .
+
+PublicationReportTypeScheme: a skos:ConceptScheme ;
+    mu:uuid "224637a8-a60a-4bcb-bd9b-f052456820a4" ;
+    skos:prefLabel "Publicatie-rapport-types" .
+
+publicationReportType:by-mandatees-on-decision-date
```

Comment:
In previous scenarios such as these, it seems to me the uuid was used as the URI, so instead of `by-mandatees-on-decision-date` (see https://github.com/kanselarij-vlaanderen/app-kaleidos/blob/development/config/migrations/20190514145609-dossier-type-codes.ttl for example)

However, I quite like that the names are more semantic in this case. `publicatieRapportType:by-mandatees-on-decision-date` says a lot more than `publicatieRapportType:6f820cf1-cd36-49fb-ae82-b36ce5dbc6cb`. So I would actually prefer it like it is, but am unsure about the policy around this. @MikiDi do you have any insight on this?

Reply:
Algemeen proberen we om zo goed mogelijk de Vlaamse uri-standaard te volgen (https://www.vlaanderen.be/digitaal-vlaanderen/onze-oplossingen/open-data/vlaamse-uri-standaard-voor-data). Die stelt
> De standaard spreekt zich verder niet uit over de concrete invulling. De interpretatie van
referentie-basis en -versie is organisatiegebonden

`by-mandatees-on-decision-date` kan je vanuit een mens-perspectief alleen maar toejuichen, maar tot nu toe doen we dat niet. Da's waarschijnlijk de `organisatie-gebonden`-kant: ook als we manueel uri's genereren, genereren we ze zoals de mu-cl-resources API ze zou genereren. Geen sterke richtlijnen dus. Ik bedenk me wel dat we toch regelmatig voortschreidend inzicht hebben over de semantische nauwkeurigheid van labels van sommige concepten. Die passen we dan aan. Wanneer een label dan al in een uri ingebakken zit, kan dat soms meer verwarring zaaien dan wat anders. In conclusie dus toch een lichte voorkeur voor "uuid-uri's"

Reply:
@MikiDi Het is me wat onduidelijk wat ik met dit soort comments moet doen. Ik herinner me dat ge leesbare URI's verkoost i.v.m. de legacy beleidsdomeinen. Is het goed om het volgende keer met UUID's te doen?